stack/building-single-package-docs.rst

```diff
@@ -28,8 +28,10 @@ This installation needs to be a recent daily or weekly build since you’ll be c
 Working from the tip of the ``main`` branch is the norm for LSST software development.
 
 Finally, the documentation build uses Documenteer_ and related Sphinx_ documentation packages.
-Documenteer_ is already installed if you are using the Rubin Conda environment (part of the usual Science Pipelines installation).
-If this is not the case, see the `Documenteer installation documentation <https://documenteer.lsst.io/pipelines/install.html>`__.
+Documenteer_ is already installed if you are using the Rubin Conda developer environment.
+If running ``package-docs --help`` fails with ``command not found``, install the developer environment with: ``mamba install rubin-env-developer``.
```

Comment:
Here's perhaps a more direct way of verifying that `rubin-env-developer` is installed:

```suggestion
You can check if ``rubin-env-developer`` is installed by running ``mamba list rubin-env``.
If not, you can install the developer environment with ``mamba install rubin-env-developer``.
```